packages/renderer/src/lib/image/ImagesList.svelte

```
@@ -226,8 +281,165 @@ async function saveSelectedImages(): Promise<void> {
+    });
+  }
+}
+
+async function checkImageForUpdates(image: ImageInfoUI): Promise<void> {
+  if (image.updateCheckInProgress || image.updateStatus) {
+    return;
+  }
+
+  const { engineId, id: imageId, tag, name } = image;
+  image.updateCheckInProgress = true;
+  images = images;
```

Biome (2.4.4)

[error] images is assigned to itself.

(lint/correctness/noSelfAssign)